backoffice/src/views/SingleBeaconRecordView.tsx

```diff
@@ -73,6 +85,26 @@ export const SingleBeaconRecordView: FunctionComponent<
   const hexId = beacon?.hexId || "";
   const numberOfUses = beacon?.uses?.length.toString() || "";
 
+  const openDialogueBox = () => {
+    setDialogueIsOpen(true);
+  };
+
+  const handleDeleteDialogueAction = async (
+    actionOptionSelected: boolean,
+    reasonForAction: string
+  ) => {
+    setDialogueIsOpen(false);
+    if (actionOptionSelected) {
+      const deleteBeaconDto: IDeleteBeaconDto = {
+        beaconId: beaconId,
+        accountHolderId: "",
```

Comment:
accountHolderId always an empty string, is it worth having it at all if it's never used from the backoffice?